core/core-workflow-steps/src/main/java/org/eclipse/sw360/antenna/jsonreader/JsonReader.java

```diff
@@ -302,6 +302,32 @@ private MatchState mapMatchState(JsonObject obj) {
         return Optional.empty();
     }
 
+    private String splitJavaScriptCoordinatesName(String key, String name) {
```

Comment:
I do not like this kind of solution where you process something multiple times, because there is no way to return two values.
Perhaps better is:
setNamespace and setName.
In this case, setNamespace could even do not set the namespace at all instead of setting it with an empty string.